MdpItem.pm

```diff
@@ -188,7 +188,7 @@ sub __handle_mdpitem_cache_setup {
         $cache_max_age = $config->get('mdpitem_max_age') || 0;
     }
 
-    my $cache_key = qq{mdpitem};
+    my $cache_key = qq{mdpitem.b};
```

Comment:
Typo? "mdpitem.b" doesn't show up anywhere else as far as I can tell

Reply:
Nope; it's just the new cache_key name --- that value gets passed back to `GetMdpItem` and used there.
(The usual ways to invalidate the cache aren't available, alas.)